client/src/extension.ts

```diff
@@ -76,6 +76,32 @@ function configToResourceSettings(
   return resourceSettings;
 }
 
+function getEnabledOnlyPaths(): EnabledOnlyPath[] {
+  const items = [] as EnabledOnlyPath[];
+  if (!vscode.workspace.workspaceFolders) {
+    return items;
+  }
+  for (const workspaceFolder of vscode.workspace.workspaceFolders) {
+    const config = vscode.workspace.getConfiguration(
+      EXTENSION_NS,
+      workspaceFolder,
+    );
+    const enabledOnlyPaths = config.get<string[]>(ENABLE_PATHS_ONLY);
+    if (!enabledOnlyPaths || !enabledOnlyPaths.length) {
+      continue;
+    }
+    const workspace = path.normalize(workspaceFolder.uri.fsPath);
+    const paths = enabledOnlyPaths.map((folder) =>
+      path.resolve(workspace, folder)
```

Comment:
Nitpick: could we do `Uri.joinPaths(workspaceFolder.uri, folder)` here instead? I believe that handles normalization and we could avoid using the "path" module.

Reply:
Ok, I thought I had run into problems with `.fsPath` not being fully normalized, but maybe that was just me being paranoid.